Assets/Scripts/Ground.cs

```diff
@@ -0,0 +1,83 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class Ground : MonoBehaviour
+{
+    [SerializeField] private Material _normalMaterial;
+    [SerializeField] private Material _highlightMaterial;
+    [SerializeField] private Material _selectedMaterial;
+
+    private Transform _thisTransform;
+    private Vector3 _position;
+    private float _startHeight = 5f;
+
+    public enum State { NORMAL, HIGHLIGHT, SELECTED };
+
+    private Renderer _renderer;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        _renderer = GetComponent<Renderer>();
+        _thisTransform = this.GetComponent<Transform>();
```

Comment:
is dit `this.GetComponent<Transform>()` niet hetzelfde als `transform`

Reply:
yes